jest-integration.config.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
-    testEnvironment: 'node',
-    roots: ['<rootDir>/integration'],
-    testMatch: ['**/*.test.ts'],
-    transform: {
-        '^.+\\.tsx?$': 'ts-jest',
-    },
+  testEnvironment: 'node',
+  roots: ['<rootDir>/integration'],
+  testMatch: ['**/SimpleCacheClient.test.ts'],
```

Comment:
Why not * anymore?

Reply:
Same as https://github.com/momentohq/client-sdk-javascript/pull/43#discussion_r832868856

Reply:
Are we tracking this somewhere?

Reply:
Good catch! I'll update it,